src/bin/pg_autoctl/keeper.c

```diff
@@ -479,11 +479,13 @@ keeper_update_pg_state(Keeper *keeper)
 	 * In some states, PostgreSQL isn't expected to be running, or not expected
 	 * to have a streaming replication to monitor at all.
 	 */
+	log_debug("keeperState->current_role = %s", NodeStateToString(keeperState->current_role));
```

Comment:
do we want to keep this debug output once the PR is merged? it seems helpful for cooking the PR, maybe not so much once we get it to being mergeable?

Reply:
removed